### DISCUSSION TITLE
Emporia Vue 3 flashing issue

### DISCUSSION THREAD

**Discussion by Alkynao** [emporia-vue-local/esphome#300] [Q&A]
Hi all,

I'm trying to flash an Emporia Vue 3 and it don't work's, i used this .yaml conf (ssid and password change):

https://digiblur.com/2024/03/14/emporia-vue-gen3-esp32-esphome-home-assistant/

![Problème](https://github.com/user-attachments/assets/688182ee-bee3-47ca-9192-406bb458c0ab)

Can you help me please...

**Comment by mattster98** [comment]
Can you compile anything?  I think this is a build environment issue.  If you search for the specific error "No module named 'pkg_resources'" there's some good info on adding setuptools to your environment.  I'd give that a try and see if it helps.  My guess is some sort of path issue on Windows if you already have it installed, but I'm not super familiar with the setup you're using.

**Comment by Alkynao** [comment]
Hi,

Sorry for the inconvenience, i simply install "setuptool" with this command line (powershell):
`pip install setuptools`

And it work's done.
